packages/universal-cookie/package.json

```diff
@@ -3,13 +3,22 @@
   "version": "7.0.0",
   "description": "Universal cookies for JavaScript",
   "main": "cjs/index.js",
+  "types": "cjs/index.d.ts",
```

Comment:
Type files are only kept in the esm folder. You need to change it to esm folder.

Reply:
Well turns out I'm wrong. It makes sense to keep it that way.